docker/ci/build-image.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Data Prepper already has a `release` directory. Perhaps this should go in there.

Reply:
I created a directory structure to match what the opensearch team is using. I can move the script to our release directory.